api/src/client/ApiTaxFilingClient.ts

```diff
@@ -49,10 +49,26 @@ export const ApiTaxFilingClient = (config: ApiConfig, logger: LogWriterType): Ta
       logger.LogInfo(
         `TaxFiling Lookup - NICUSA - Id:${logId} - Response received: ${JSON.stringify(response.data)}`
       );
+
       const apiResponse = response.data as ApiTaxFilingLookupResponse;
+
+      const city = apiResponse.Data?.find((x) => {
+        if (x.Name === "tax-city") {
```

Comment:
Let's avoid using `x` as a variable and pick something else. Even a little more descriptive like `item` would be preferable to a single character.